_posts/0000-01-14-apay1.md

```diff
@@ -5,5 +5,7 @@ title: "Welcome to our slide deck!"
 
 Caption here
```

Comment:
You didn't replace the caption and put the caption here where it says "caption here" :/